hermes-ffi/src/ontology/injection.rs

```diff
@@ -386,6 +386,72 @@ impl AsRust<hermes::InjectionResetCompleteMessage> for CInjectionResetCompleteMe
     }
 }
 
+#[repr(C)]
+#[derive(Debug)]
+pub struct CInjectionFailedMessage {
+    pub request_id: *const libc::c_char,
```

Comment:
this field is nullable, add at least a `/// nullable` comment on it so that this appears in the generated .h